### ISSUE TITLE
Twig calls invalid is function

### ISSUE THREAD

**Issue by alexander-schranz** [twigphp/Twig#2278]
On my update from `1.27` -> `1.28` my application did crash.

I think the change come from: `Fixed precedence between getFoo() and isFoo() in Twig_Template::getAttribute()`

I have the following in my twig template:

```twig
{{ thread.creator.username }}
```

but now instead of `getCreator` it is calling `isCreator` which look this way in my case:

```php
    public function isCreator(UserInterface $user)
    {
        return $this->creator->getId() === $user->getId();
    }
```

and ends in a 500 error `Type error: Argument 1 passed to AppBundle\Entity\Thread::isCreator() must implement interface Symfony\Component\Security\Core\User\UserInterface, none given`.

I know I can do `getCreator()` but behaviour change  is definitely a BC Break and I think I'm not the only one with this problem.

**Comment by jvasseur** [comment]
The bug should be fixed in twig 1.28.2, it was fixed by #2272

**Comment by alexander-schranz** [comment]
@jvasseur thank you! did update to 1.28.2 it is fixed.
